recipes/grpc/all/conandata.yml

```diff
@@ -1,4 +1,16 @@
 sources:
+  "1.42.0":
+    url: "https://github.com/grpc/grpc/archive/refs/tags/v1.42.0.tar.gz"
+    sha256: "b2f2620c762427bfeeef96a68c1924319f384e877bc0e084487601e4cc6e434c"
+  "1.41.1":
+    url: "https://github.com/grpc/grpc/archive/refs/tags/v1.41.1.tar.gz"
+    sha256: "12a4a6f8c06b96e38f8576ded76d0b79bce13efd7560ed22134c2f433bc496ad"
+  "1.41.0":
+    url: "https://github.com/grpc/grpc/archive/refs/tags/v1.41.0.tar.gz"
+    sha256: "e5fb30aae1fa1cffa4ce00aa0bbfab908c0b899fcf0bbc30e268367d660d8656"
+  "1.40.0":
+    url: "https://github.com/grpc/grpc/archive/refs/tags/v1.40.0.tar.gz"
+    sha256: "13e7c6460cd979726e5b3b129bb01c34532f115883ac696a75eb7f1d6a9765ed"
```

Comment:
Are these intermediate versions required by some project?
We usually only add the most recent version, and add intermediate (older) releases only when needed.

Reply:
Yes, we'd like to have these too if possible from Conan.

Reply:
@madebr , why is it a problem? Longer CI times?

Reply:
Longer CI times only apply when modifying the py recipe. (the CI is smart enough to not rebuild everything when e,g. only conandata.yml changes).
My main concern with blanket addition of versions is the maintainability. Sometimes this requires us to maintain multiple patches.
But perhaps this is not the case with grpc as you were able to re-use the same patch for all versions.

Reply:
Understood :-) Let's give it a try.. These seem to be pretty compatible and we cover way more use cases and projects.